vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -563,6 +563,16 @@ void TextInputShadowNode::updateProperties(const folly::dynamic &&props) {
             } else if (propertyValue.isNull())
               textBox.ClearValue(winrt::TextBox::TextProperty());
           }
+        } else if (propertyName == "autoCapitalize") {
+          if (propertyValue.isString()) {
+            if (propertyValue.asString() == "characters") {
+              textBox.CharacterCasing(winrt::CharacterCasing::Upper);
```

Comment:
>CharacterCasing [](start = 22, length = 15)

Please pay attention, CharacterCasing is introduced since 16299 but vnext supports 15063

Reply:
Thanks, I forgot about that this was a recent supported property.

---
In reply to: [334735165](https://github.com/microsoft/react-native-windows/pull/3406#discussion_r334735165) [](ancestors = 334735165)